data/postRouter.js

```diff
@@ -64,10 +64,38 @@ router.put('/:id', (req, res) => {
             res.status(200).json(updated)
         }
     }).catch(error => {
-        res.status(500).json({ message: 'The post information could not be modified.' })
+        res.status(500).json({ error: 'The post information could not be modified.' })
+    })
+})
+
+router.get('/:id/comments', (req, res) => {
+    const id = req.params.id;
+    
+    Posts.findPostComments(id).then(comments => {
+        if(comments && comments.length){
+            res.status(200).json(comments)
+        } else {
+            res.status(404).json({ message: "The post with the specified ID does not exist." })
```

Comment:
This would fire if a post has no comments yet, to check if a post exists you would have to use findPostById before findPostComments